requirements.txt

```diff
@@ -1,32 +1,7 @@
-# Non RobotPy packages, some are required by RobotPy because for simplicity we don't pin them,
-# just explicitly list that they are required
-attrs
-astroid
-bcrypt
-cffi
-click
-cryptography
-lazy-object-proxy
-iniconfig
-isort
-mccabe
-pluggy
-packaging
-paramiko
-pint
-platformdirs
-py
-pycparser
-pynacl
-pyparsing
-pyyaml
+# Non RobotPy packages that we use
+black
 pylint
 pytest
-six
-toml
-typing-extensions
-wheel
```

Comment:
I don't think this is automatically available on some platforms (windows?) which can lead to long setup times if robotpy has to fall back to a non-wheels install (this may be me misremembering though or latest robotpy may mandate wheels-only install).
If I am right though, we probably want to keep this in.

Reply:
Good point, I'm not sure wheel specifically is needed, though we definitely need the upgraded version of pip, but you can't specify that in the requirements file since it won't restart pip until the install completes.

But I actually wanted to run something against windows to provide some additional sanity check.  I can't run the simulator but I could run pylint on windows which would at least verify we can install the packages and do "something"